float64 q_sd
float64 q_n
float64 q_mu
float64 r_dels
float64 r_ax